src/displayPokemon.js

```diff
@@ -1,27 +1,33 @@
+import { postComment, getComments } from '../modules/involvementRequests.js';
+import displayMessage from './displayMessage.js';
 /* eslint-disable no-new */
 const cardPoke = document.querySelector('.cardModal');
 const $grafica = document.querySelector('#grafica');
 let nroAbility = 0;
 let datosIngresos = {};
```

Comment:
These variables should be inside the displayPokemon() function so they can work adequately 😉.

Reply:
Thank you for your feedback, I made the changes